lib/scripts/clone.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+set -ex #TODO x #ONLY FOR DEV
```

Comment:
only `set -e` in production.
You can use something like (untested): ` [ -n "$ENV"  -a "$ENV" != 'production' ] && set -x` and set ENV according `PackageProvider.env` which will set form ENV['ENV'] || ENV['RAILS_ENV'] || ENV['RACK_ENV'] || 'development'